string command_id
string condition
---
string results